R/utils_insert_db.R

```diff
@@ -294,14 +294,14 @@ rescore_package <- function(pkg_name,
   
   db_table <- dbSelect("SELECT metric.name, package_metrics.encode FROM package 
                        INNER JOIN package_metrics ON package.id = package_metrics.package_id
-                       INNER JOIN metric ON package_metrics.metric_id = metric.id
+                       INNER JOIN metric ON package_metrics.id = metric.id
```

Comment:
Why was this changed? `metric_id` is the foreign key from the `metric` table in the `package_metrics` table.

Reply:
Because `metric_id` was all NA and it wasn't working for me.